libs/hiraya/src/components/Button/index.tsx

```diff
@@ -1,5 +1,58 @@
-const Button = () => {
-  return <button>Hello World</button>;
+import { Button as MUIButton, styled } from '@mui/material';
+
+interface ButtonProps {
+  label?: string;
+  variantStyle?: 'primary' | 'secondary' | 'tertiary';
+}
+
+const ButtonStyled = styled(MUIButton)<ButtonProps>(({ variantStyle }) => ({
+  backgroundColor:
```

Comment:
What if instead of doing nested ternary operators you do an object map instead like this, in which we do an exhaustive pattern

```ts
const backgroundColor = {
  primary: "#305EFF'",
  secondary: '#FFFFF',
  tertiary: '#BDBDBD'
}

// can be used like this
backgroundColor: backgroundColor(variantStyle);
```